starter-code/index.html

```diff
@@ -3,40 +3,82 @@
 <head>
   <meta charset="UTF-8">
   <title>Spotify</title>
+  
+  <link href="https://fonts.googleapis.com/css?family=Roboto:300,400,400i,500,700" rel="stylesheet">
+  
+
+  <link rel="stylesheet" href="./css/main.css">
 </head>
+
 <body>
-Premium Discover Help Download
-
-Music for everyone.
-Spotify is now free on mobile, tablet and computer.
-Listen to the right music, wherever you are.
-
-What’s on Spotify?
-
-Millions of Songs
-There are millions of
-songs on Spotify
-
-HD Music
-Listen to music as if you
-were listening live
-
-Stream Everywhere
-Stream music on your
-smartphone, tablet or
-computer
-
-It’s as yeezy as Kanye West.
-Search
-Know what you want to listen to?
-Just search and hit play.
-Browse
-Check out the latest charts,
-brand new releases and great
-playlists for right now.
-Discover
-Enjoy new music every Monday
-with your own personal playlist.
-Or sit back and enjoy Radio.
+    <nav class="web-nav">
+      <div>
+        <a href="#0"><img class="logo" src="./images/spotify-logo.png" alt=""></a>
```

Comment:
adding alt tags to images is important for accessibility. here, you could just call it "Spotify logo"